test/controllers/CYMinusOneLiabilityControllerSpec.scala

```diff
@@ -43,8 +43,8 @@ class CYMinusOneLiabilityControllerSpec extends SpecBase with MockitoSugar {
   def form(arguments: Seq[Any]) = formProvider.withPrefix("cyMinusOne.liability", arguments)
 
   val fullDatePattern: String = "d MMMM yyyy"
-  val taxYearStart: String = "6 April 2019"
-  val taxYearEnd: String = "5 April 2020"
+  val taxYearStart: String = TaxYearRange(CYMinus1TaxYear).startYear
```

Comment:
Should this be:
val taxYearStart: String = TaxYearRange(CYMinus2TaxYear).startYear